news-analysis.py

```diff
@@ -93,7 +93,7 @@
 MINUMUM_ARTICLES = 1
 
 # define how often to run the code (check for new + try to place trades)
-# in minutes
+# in seconds
```

Comment:
This should be in minutes, as you're multiplying this by 60 in the execution of the code

Reply:
it was changed to 1....in line 473....now it's back to 60 after your commit :-)

Reply:
Re the first issue, it was also reported by another user. Not only on my machine...perhaps a timeout that we received due to slower network.